CodeMaid/UI/Dialogs/Options/Cleaning/CleaningUpdateDataTemplate.xaml

```diff
@@ -14,56 +16,63 @@
             </GroupBox>
             <GroupBox Header="{x:Static p:Resources.FileHeader}">
                 <StackPanel>
-                    <TabControl>
+                    <TabControl x:Name="LanguageTab" SelectedIndex="{Binding SelectedTabIndex}">
```

Comment:
The name `LanguageTab` can be removed.  Since we're using MVVM I try to avoid names so there isn't the temptation to directly manipulate view elements from the view model.